pkg/api/types.go

```diff
@@ -44,13 +44,21 @@ type DeschedulerStrategy struct {
 	Params *StrategyParameters
 }
 
+// Namespaces carries a list of included/excluded namespaces
+// for which a given strategy is applicable
+type Namespaces struct {
+	Include []string
+	Exclude []string
```

Comment:
Should note that these can't both be set (is that right? How do you interpret setting both include and exclude?)

Reply:
Added validation in every strategy for which the namespace filtering applies